frontend/src/components/Sidebar/ListItemLink.tsx

```diff
@@ -24,11 +27,29 @@ interface ListItemLinkProps {
   pathname: string;
   search?: string;
   name: string;
+  subtitle?: string;
   icon?: IconProps['icon'];
+  iconOnly?: boolean;
+  containerProps?: {
+    [prop: string]: any;
```

Comment:
Can `any` be improved here?

Reply:
This is spread in a `li` component. Is there a good type there?